src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -270,9 +235,43 @@ public abstract partial class JsonNode
         public abstract System.Text.Json.JsonNode Clone();
         public static System.Text.Json.JsonNode DeepCopy(System.Text.Json.JsonElement jsonElement) { throw null; }
         public static System.Text.Json.JsonNode GetNode(System.Text.Json.JsonElement jsonElement) { throw null; }
-        public static System.Text.Json.JsonNode Parse(string json) { throw null; }
+        public static implicit operator System.Text.Json.JsonNode (bool value) { throw null; }
```

Comment:
What about nullable types?

```csharp
bool? nullableBool = true;
JsonToken token = nullableBool;
```

Should have implicit operator for all structs:
```csharp
public static implicit operator System.Text.Json.JsonNode (bool? value) { throw null; }
```

Reply:
Let's consider that separately (worth noting in the doc).